src/composable/Questionnaire.ts

```diff
@@ -172,19 +173,22 @@ export class Questionnaire implements IQuestionnaire {
    * @returns
    */
   getProgressPercent(props: IStepData, config: IQuestionableConfig): number {
+    if (matches(props.step?.sectionId, PAGE_TYPE.RESULTS)) {
+      return 100;
+    }
     const sections = this.getSections(props, config);
     const lastStep = sections[sections.length - 1];
     // if there is no step, the questionnaire has just started
     if (!lastStep?.lastStep) {
       // Always show a little progress
-      return 1;
+      return 0.5;
     }
     const thisStepIdx = this.flow.indexOf(`${props.stepId}`);
     // add 2 to account for the summary and result steps
     let lastStepIdx = lastStep.lastStep + 2;
     if (config.mode === MODE.EDIT) {
       // if in design mode, every step will be iterated
-      lastStepIdx = this.flow.length;
+      lastStepIdx = this.flow.length - 1;
     }
     // To calculate the percent, divide the index of this step
     //   by the index of the last step multiplied by 100.
```

Comment:
Oh, the result is 1-100 but internally the number is 0-1.0?
Line 184 is returning 0.5... so I'm confused.